cdc/model/changefeed.go

```diff
@@ -584,3 +584,15 @@ func (t DownstreamType) String() string {
 	}
 	return "Unknown"
 }
+
+// ChangeFeedStatusForAPI uses to transfer the status of changefeed for API.
+type ChangeFeedStatusForAPI struct {
```

Comment:
It is better to avoid creating duplicate entities here.

Reply:
It is the best way I can find now...